### PR TITLE
[Presence Update] 1.0.1 Glimesh

### PR DESCRIPTION
**Pull request by ItzNxthaniel**
- Blogs Section of Website got moved to a Subdomain so fixed that.
  - Added the new About Page, Alpha Features.
  - Changed Logic of getting Blog Post Title.

### CODE REVIEW

**Comment by ItzNxthaniel** [comment]
Err, think the checks broke.

**Comment by ririxidev** [comment]
Obv it did bc you have conflicting files :bruh:

**Comment by ItzNxthaniel** [comment]
> Obv it did bc you have conflicting files :bruh:

They just updates ;-;

**Comment by ItzNxthaniel** [comment]
> Obv it did bc you have conflicting files :bruh:

Sorry, lil dumb. Barely use PRs :)

**Comment by EncryptedDev** [comment]
screenshots of proof of working?

**Comment by ItzNxthaniel** [comment]
> screenshots of proof of working?
Mind showing where it says I need to supply that? Never had to provide them with the first PR I did for this presence.

**Comment by ItzNxthaniel** [comment]
image
image
image

Only things I added :)

**Review by callumok2004 — CHANGES_REQUESTED** [review]
Bump version in `metadata.json`

**Comment by ItzNxthaniel** [comment]
> Bump version in `metadata.json`

Already changed... It was changed in the first commit.

**Review by Bas950 — CHANGES_REQUESTED** [review]
Bump version

**Comment by ItzNxthaniel** [comment]
> Bump version

What version y'all want? I already bumped it from 1.0.0 to 1.0.1

**Comment by EncryptedDev** [comment]
> > Bump version
> 
> What version y'all want? I already bumped it from 1.0.0 to 1.0.1

yes

**Comment by ItzNxthaniel** [comment]
> > > Bump version
> > 
> > 
> > What version y'all want? I already bumped it from 1.0.0 to 1.0.1
> 
> yes

Okay, so I've already bumped it? Check the initial commit.

**Comment by ItzNxthaniel** [comment]
> > > Bump version
> > 
> > 
> > What version y'all want? I already bumped it from 1.0.0 to 1.0.1
> 
> yes

The version is already bumped in the initial commit, da7ccd9. Is there something else I may be missing?

**Comment by VocalZero** [comment]
Your Presence is currently on version 1.0.1 publicly available on PreMiD. You need to change the version to 1.0.2 now.

**Comment by ItzNxthaniel** [comment]
> Your Presence is currently on version 1.0.1 publicly available on PreMiD. You need to change the version to 1.0.2 now.

I see, I didn't realized my code was refactored and that the version was bumped. Complete issue on my side for not pulling before I started work on the 1.0.2 update. Sorry about that. Fixed in cd98825

File: websites/G/Glimesh/presence.ts [hunk 1]
```diff
@@ -166,6 +142,26 @@ presence.on("UpdateData", async () => {
...
+  } else if (locationHost == "blog.glimesh.tv") {
+    presenceData.details = "Viewing Blogs";
+
+    if (locationPath.match("/posts")) {
+      const blogPost = document.querySelector("body > div > div > div > h1 > a").textContent;
+      
+      presenceData.details = "Reading" + (!await presence.getSetting("show_details") ? " a " : " ") + "Blog";
+      if (await presence.getSetting("show_details")) {
+        presenceData.state = blogPost;
+      }
```
Comment (ririxidev) [inline websites/G/Glimesh/presence.ts:154]:
```suggestion
      if (await presence.getSetting("show_details")) presenceData.state = blogPost;
```

**Review by ririxidev — CHANGES_REQUESTED** [review]

File: websites/G/Glimesh/presence.ts (hunk 1 above)
Comment (ItzNxthaniel) [reply websites/G/Glimesh/presence.ts:154]: Just those lines or do you want me to replicate that through out the entire script?

Comment (ririxidev) [reply websites/G/Glimesh/presence.ts:154]: Yes.

**Review by ririxidev — APPROVED** [review]

**Review by Bas950 — APPROVED** [review]

**Pull request merged**